src/dotnet/UI.Blazor/Components/ContextMenu/context-menu.css

```diff
@@ -2,3 +2,15 @@
     @apply fixed;
     z-index: 200;
 }
+
+.context-menu-old-button {
```

Comment:
Wierd name 'context-menu-old-button'. Why do we need 'old' suffix?
Do we still have several ways to define context menu?

Reply:
oops.. i forgot to rename it